orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/virus/VirusInterpreterEntry.java

```diff
@@ -30,6 +30,8 @@ public interface VirusInterpreterEntry
 
     boolean reported();
 
+    boolean blacklisted();
+
```

Comment:
Why did you decide to add this property if you remove blacklisted virus entries anyways? 

In other words, the only producer of ORANGE datamodel guarantees this property is false - maybe cleaner to just remove it?

Reply:
Yes you are totally correct, doesn't make sense!